lib/SPIRV/SPIRVRegularizeLLVM.h

```diff
@@ -51,10 +51,10 @@ class SPIRVRegularizeLLVMBase {
   // Lower functions
   bool regularize();
 
-  // SPIR-V disallows functions being entrypoints and called
-  // LLVM doesn't. This adds a wrapper around the entry point
-  // that later SPIR-V writer renames.
-  void addKernelEntryPoint(llvm::Module *M);
+  // SPIR-V disallows functions being entrypoints/kernels and called
+  // OpenCL doesn't. This adds a wrapper around the entry point if it's called
```

Comment:
Strictly speaking the part "if it's called by other entry point" isn't handled by this function anymore?

Reply:
Yeah, I've rephrased it a bit, I still want to keep 'if it's called by another entry point text here'